diktat-rules/src/main/kotlin/org/cqfn/diktat/ruleset/rules/LineLength.kt

```diff
@@ -74,6 +109,191 @@ class LineLength : Rule("line-length") {
         }
     }
 
+    private fun fixError(wrongNode: ASTNode, configuration: LineLengthConfiguration, grand: ASTNode) {
+        if (wrongNode.elementType == EOL_COMMENT) {
+            fixComment(wrongNode, configuration)
+            return
+        }
+        var parent = wrongNode
+        do {
+            when (parent.elementType) {
+                FUN -> {
+                    if (!parent.hasChildOfType(BLOCK))
+                        parent.addChild(PsiWhiteSpaceImpl("\n"), parent.findChildByType(EQ)!!.treeNext)
+                    return
+                }
+                CONDITION -> {
+                    fixCondition(parent, configuration, grand)
+                    return
+                }
+                PROPERTY -> {
+                    fixProperty(parent, grand, configuration)
+                    return
+                }
+                else -> parent = parent.treeParent
+            }
+        } while (parent.treeParent != null)
+    }
+
+    private fun fixComment(wrongNode: ASTNode, configuration: LineLengthConfiguration){
+        var node = wrongNode
+        do {
+            val commentText = node.text
+            var indexLastSpace = configuration.lineLength.toInt()
+            while (commentText[indexLastSpace] != ' ') {
+                --indexLastSpace
+                if (indexLastSpace == 0)
+                    return
+            }
```

Comment:
```suggestion
    val indexLastSpace = node.text.substring(0, configuration.lineLength).lastIndexOfOrNull(' ') ?: return
```
Maybe, something like this?